verified-broadcast/tests/broadcast.test.js

```diff
@@ -0,0 +1,43 @@
+const broadcastFunction = require('../functions/broadcast').handler;
+const helpers = require('../../test/test-helper');
+
+const mockService = {
+  verificationChecks: {
+    create: jest.fn(() =>
+      Promise.resolve({
+        sid: 'my-new-sid',
+      })
+    ),
```

Comment:
Just a tip to simplify this, I think you can use [`mockResolvedValue`](https://jestjs.io/docs/mock-function-api#mockfnmockresolvedvaluevalue) here, like:

```js
const mockService = {
  verificationChecks: {
    create: jest.fn().mockResolvedValue({ sid: "my-new-sid" })
  }
};
```

Reply:
thank you!